answers/final.py

```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict, Counter
 
-from gensim.parsing.preprocessing import STOPWORDS
-from gensim.utils import simple_preprocess
 import networkx
-import nltk
```

Comment:
if you like, you can also remove it from https://github.com/coala/corobo/blob/master/answers/requirements.txt

Reply:
Created new issue about that https://github.com/coala/corobo/issues/593 , for another newcomer to complete